### PR TITLE
add RELEASE_NOTES optional parameter to the create-a-release job

### PR DESCRIPTION
**Pull request by arturopala**

### CODE REVIEW

**Comment by arturopala** [comment]
Tested successfuly here: https://github.com/hmrc/agent-epaye-registration/releases/tag/v0.13.0

**Comment by arturopala** [comment]
example of handling double-quotes escaping https://github.com/hmrc/agent-epaye-registration/releases/tag/v0.15.0

File: src/test/groovy/uk/gov/hmrc/jenkinsjobs/JobsSpec.groovy [hunk 1]
```diff
@@ -12,7 +12,7 @@ class JobsSpec extends Specification {
     @Unroll
     void 'test script #file.name'(File file) {
         when:
-        DslScriptLoader.runDslEngine file.text, new MemoryJobManagement()
+        new DslScriptLoader(new MemoryJobManagement()).runScript file.text
```
Comment (arminio) [inline src/test/groovy/uk/gov/hmrc/jenkinsjobs/JobsSpec.groovy]: Can't make sense of this test change. @arturopala Can you please me a brief on this?

Comment (arturopala) [reply src/test/groovy/uk/gov/hmrc/jenkinsjobs/JobsSpec.groovy]: it's consequence of jenkins-job-builders version update

Comment (arminio) [reply src/test/groovy/uk/gov/hmrc/jenkinsjobs/JobsSpec.groovy]: 👍

**Pull request merged**